_layouts/software-details.html

```diff
@@ -74,7 +74,10 @@ <h3 class="color-content font-serif my-2 my-md-4">
               <p><span class="label">{{ t.software.technical_contact }} </span>
                 {% for contact in page.publiccode.maintenance.contacts limit: 1 %}
                     {% if contact.email != nil and contact.email != "" %}
-                    <a href="mailto:{{ contact.email }}">{{ contact.name }}</a>
+                    <a href="mailto:{{ contact.email }}">
+                      <span class="icon it-mail d-inline-block"></span>
```

Comment:
I wouldn't introduce this change in this PR since it's not consistent with the rest of the catalogue. If we decide to go the hard way and switch all external links to use such icon we might do it consistently in a separate issue + PR.